scripts/build/build/targets.py

```diff
@@ -620,6 +618,12 @@ def BuildBouffalolabTarget():
     target.AppendModifier('cdc', enable_cdc=True)
     target.AppendModifier('resetCnt', enable_resetCnt=True)
     target.AppendModifier('rotating_device_id', enable_rotating_device_id=True)
+    target.AppendModifier('mfd', function_mfd="release")
+    target.AppendModifier('mfdtest', function_mfd="test")
+    target.AppendModifier('ethernet', enable_ethernet="test")
```

Comment:
enable_ethernet is defined as "bool" same with other bits here. I am guessing some copy&paste is going on here... please fix.

Reply:
Corrected.